File: 3_cppl/SingleThread.cpp
#include<iostream>
#include<thread>
using namespace std;

void func1(){
    cout<< "Hello World!" << endl;
}

int main(){
    thread t1(func1);
    t1.join();
    thread t2 (func1);
    t2.join();
    
    return 0;
}

File: 3_cppl/SingleThread.java
class Single extends Thread{
    public void run(){
        System.out.println("Hello World!");
    }
}
public class SingleThread{
    public static void main(String[]args){
        Single t1= new Single();
        Single t2= new Single();
        t1.start();
        t2.start();
    }
}
